src/app/showcase/panel/common/LinksViewer.tsx

```diff
@@ -1,43 +1,56 @@
-import { Stack } from '@mui/material'
+import { Stack, Typography } from '@mui/material'
 import React from 'react'
 import LabelWithIcon from '../LabelWithIcon'
 import { TagIcon } from '@/icons'
 import * as Style from './SkillInput.style'
 
 interface IlinksProps {
-  links: any
+  links: string[] | undefined
 }
 
 const LinksViewer = ({ links }: IlinksProps) => {
+  const convertLink = (link: string) => {
+    const httpPattern = /^https?:\/\//i
+    if (!httpPattern.test(link)) {
+      return `http://${link}`
+    }
+    return link
+  }
+
   return (
-    <Stack>
-      <LabelWithIcon
-        svgIcon={<TagIcon sx={Style.IconStyle} />}
-        message="링크모음"
-      />
-      {links?.map((link: any) => {
-        return (
-          <Stack
-            direction={'column'}
-            spacing={'0.5rem'}
-            width={'26rem'}
-            key={link.id}
-          >
-            <Stack
-              spacing={0.75}
-              py={1}
-              flexWrap={'wrap'}
-              width={1}
-              direction={'row'}
-              useFlexGap
-            >
-              <a href={link.link} target="_blank" rel="noreferrer">
+    <Stack
+      sx={{
+        gridArea: 'linksViewer',
+        width: '100%',
+        height: '100%',
+        gap: '0.75rem',
+      }}
+    >
+      <Stack>
+        <LabelWithIcon
+          svgIcon={<TagIcon sx={Style.IconStyle} />}
+          message="링크모음"
+          color="text.alternative"
+        />
+      </Stack>
+      <Stack>
+        {links?.map((link: any) => {
+          return (
+            <>
+              <Typography
+                component="a"
+                color={'text.normal'}
+                variant="body2"
```

Comment:
디자이너님이 지정해주신 위계는 모두 대문자로 시작합니다..!
```suggestion
                variant="Body2"
```